File: Python/PRAK205-2310817310004-MSAMILRENDYNORSALEH.py
import math

a, b = map(int, input("").split())
c = math.sqrt(b**2 - a**2)
tinggi = a
keliling = a + b + c
luas = 0.5 * c * a
print(f"Alas = {c:.0f} cm")
print(f"Tinggi = {tinggi} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm^2")

File: C/PRAK205-2310817310004-MSAMILRENDYNORSALEH.c
#include <stdio.h>
#include <math.h>

int main() {
    double a, b;
    printf("");
    scanf("%lf %lf", &a, &b);
    float c = sqrt(b * b - a * a);
    float tinggi = a;
    float keliling = a + b + c;
    float luas = 0.5 * c * a;
    printf("Alas = %.0lf cm\n", c);
    printf("Tinggi = %.0lf cm\n", tinggi);
    printf("Keliling = %.0lf cm\n", keliling);
    printf("Luas = %.0lf cm^2\n", luas);

    return 0;
}
